core/src/SYCL/Kokkos_SYCL_Space.cpp

```diff
@@ -66,14 +66,10 @@ void USM_memcpy(Kokkos::Experimental::Impl::SYCLInternal& space, void* dst,
 
 void USM_memcpy(void* dst, const void* src, size_t n) {
   Experimental::SYCL().fence();
-  try {
-    USM_memcpy(*Kokkos::Experimental::Impl::SYCLInternal::singleton().m_queue,
-               dst, src, n)
-        .wait_and_throw();
-  } catch (sycl::exception const& e) {
-    Kokkos::Impl::throw_runtime_exception(
-        std::string("There was a synchronous SYCL error:\n") + e.what());
-  }
+  auto event =
+      USM_memcpy(*Kokkos::Experimental::Impl::SYCLInternal::singleton().m_queue,
```

Comment:
Be consistent within the body of that function
```suggestion
      USM_memcpy(*Experimental::Impl::SYCLInternal::singleton().m_queue,
```